### PR TITLE
MCKIN-11026 Fixed item position

### PR DESCRIPTION
**Pull request by nasirhjafri**

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @nasirhjafri! I've created [OSPR-3745](https://openedx.atlassian.net/browse/OSPR-3745) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: drag_and_drop_v2/public/js/drag_and_drop.js [hunk 1]
```diff
@@ -1456,11 +1456,10 @@ function DragAndDropBlock(runtime, element, configuration) {
             var container_height = $container.innerHeight();
 
             // We need to get the item position relative to the $container.
-            var item_offset = $item.offset();
             var container_offset = $container.offset();
             var original_position = {
-                left: drag_origin.x - container_offset.left,
-                top: item_offset.top - container_offset.top
+                left: drag_origin.x - container_offset.left - 50,
```
Comment (msaqib52) [inline drag_and_drop_v2/public/js/drag_and_drop.js]: why subtract `50`?

Comment (nasirhjafri) [reply drag_and_drop_v2/public/js/drag_and_drop.js]: On request from front-end team.
cc: @shahidrafique

Comment (shahidrafique) [reply drag_and_drop_v2/public/js/drag_and_drop.js]: because we have 100px fixed size box.

Comment (msaqib52) [reply drag_and_drop_v2/public/js/drag_and_drop.js]: @nasirhjafri rather than using this magic number, can we add it to constant and use it here.

**Pull request merged**